README.prod.md

```diff
@@ -37,35 +37,31 @@ Installing Zulip in production
 These instructions should be followed as root.
 
 (1) Install the SSL certificates for your machine to
-  /etc/ssl/private/zulip.key
-  and
-  /etc/ssl/certs/zulip.combined-chain.crt
+    `/etc/ssl/private/zulip.key` and `/etc/ssl/certs/zulip.combined-chain.crt`
 
-(2) download zulip-server.tar.gz, and unpack to it /root/zulip, e.g.
+(2) Download `zulip-server.tar.gz`, and unpack to it `/root/zulip`, e.g.
+```
 tar -xf zulip-server-1.1.3.tar.gz
 mv zulip-server-1.1.3 /root/zulip
+```
 
-(3) run /root/zulip/scripts/setup/install
-
-This may take a while to run, since it will install a large number of
-packages via apt.
+(3) run `/root/zulip/scripts/setup/install`.
```

Comment:
It's probably worth having the actual commands to run sit inside a block code section so that if one is reading this in a terminal environment, you can just copy-paste a section of text rather than needing to highlight particular parts of lines.